src/04-arrays.js

```diff
@@ -24,7 +25,10 @@ function createEmptyArray() {}
  *  createArrayWithTwoElements(true, false);
  *  //> [ true, false ]
  */
-function createArrayWithTwoElements(a, b) {}
+function createArrayWithTwoElements(a, b) {
+newArray = new Array(a,b);
```

Comment:
Interesting use of the array initializer here! Remember we could have also just put these values in an array literal: `[a, b]`